openshift-ci/Dockerfile.bundle.ci

```diff
@@ -11,4 +11,5 @@ LABEL operators.operatorframework.io.bundle.package.v1=performance-addon-operato
 LABEL operators.operatorframework.io.bundle.channels.v1=4.6.0
 LABEL operators.operatorframework.io.bundle.channel.default.v1=4.6.0
 
-COPY ${OLM_SOURCE}/performance-addon-operator/${CHANNEL}.0/ /
+COPY deploy/metadata/performance-addon-operator/${CHANNEL}.${ZVERSION}/* /metadata/
```

Comment:
could you please elaborate why this is needed? our CI is already producing manifests bundle, so is not clear to me why this needs to be fixed:
https://quay.io/repository/openshift-kni/performance-addon-operator-bundle